### DISCUSSION TITLE
Default draw.io configuration

### DISCUSSION THREAD

**Discussion by ch-westphal** [jgraph/drawio#4652] [Q&A]
I'm looking to simplify the use of draw.io considerably so that I can use it with young schoolchildren to make electrical diagrams.
With the URL parameters I've arrived at : 

- remove the splash screen (splash=0) 
- set the language (lang=fr)
- impose local storage (local=1)
- load my personal library (clibs=Uhttps%3A%2F%2Ffoo.bar%2Fpersoelec.xml)

but I can't manage to impose, by default and without user action, connectors without any arrows, either at the beginning or at the end.
It's certainly possible to draw a connector, select it, remove the arrow and then click on "set as default style" but I'd like this to be the setting for all connectors from start-up and without any action from the user. 

**Is this possible with a URL parameter?**

If not, **is it possible by modifying your own draw.io instance?**

Thanks

Translated with DeepL.com (free version) my english is poor  :-(

**Comment by davidjgraph** [comment]
Sur https://www.drawio.com/doc/faq/configure-diagram-editor trouve "defaultEdgeStyle". Je pense que le default edge style est "endArrow=classic;html=1;rounded=0;". Supprime endArrow, i.e. utilise "html=1;rounded=0;" pour defaultEdgestyle.

**Reply by ch-westphal** [reply]
Thanks, but where should I make the change?
